src/imgui/ImGuiDisassembly.cc

```diff
@@ -179,18 +179,25 @@ static void toggleBp(uint16_t addr, const BpLine& bpLine, std::span<BreakPoint>
 		addBp->setCondition(toTclExpression(getCurrentSlot(cpuInterface, debugger, addr)));
 	}
 }
+
+static std::optional<uint16_t> selectedAddr = {};
+
```

Comment:
This should be a member variable of the `ImGuiDisassembly` class. Not a global variable.

For example now if you have more than one disassembly view open at the same time, keyboard navigation doesn't work anymore (`selectedAddr` is always reset).